packages/mobile/src/app/saga.ts

```diff
@@ -41,8 +41,8 @@ export function* appInit() {
 
   if (isDeprecated) {
     Logger.warn(TAG, 'App version is deprecated')
-    navigate(Screens.UpgradeScreen)
-    return
+    // navigate(Screens.UpgradeScreen)
```

Comment:
We probably don't want this commented out :)

Reply:
😅 fixed